src/Compilers/VisualBasic/VbcCore/VbcCore.csproj

```diff
@@ -15,7 +15,7 @@
     <LargeAddressAware>true</LargeAddressAware>
     <StartupObject>Microsoft.CodeAnalysis.VisualBasic.CommandLine.Program</StartupObject>
     <SolutionDir Condition="'$(SolutionDir)' == '' OR '$(SolutionDir)' == '*Undefined*'">..\..\..\..\</SolutionDir>
-    <OutDir>$(OutDir)core-clr\</OutDir>
+    <OutDir>$(OutDir)$(CoreClrTargetFolder)</OutDir>
```

Comment:
There are OS-specific native bits, CoreRun being the obvious one (versus CoreRun.exe for Windwos). On Linux there are ".so" files for all the native binaries. They don't show up if you build for Windows. And vice-versa.